tests/org/intellij/erlang/refactoring/ErlangSafeDeleteTest.java

```diff
@@ -18,9 +18,9 @@
 
 import com.intellij.psi.PsiElement;
 import com.intellij.refactoring.safeDelete.SafeDeleteHandler;
-import com.intellij.testFramework.fixtures.LightCodeInsightFixtureTestCase;
+import com.intellij.testFramework.fixtures.LightJavaCodeInsightFixtureTestCase;
 
-public class ErlangSafeDeleteTest extends LightCodeInsightFixtureTestCase {
+public class ErlangSafeDeleteTest extends LightJavaCodeInsightFixtureTestCase {
```

Comment:
We shouldn't use LightJavaCodeInsightFixtureTestCase, better to find a language-agnostic test case.

Reply:
my wrong using BasePlatformTestCase